crates/cxx-qt-gen/src/parser/mod.rs

```diff
@@ -102,7 +103,12 @@ impl Parser {
         }
 
         // Create a new module using only items that are not CXX-Qt items
-        module.content = Some((Brace::default(), others));
+        if !others.is_empty() {
+            module.content = Some((Brace::default(), others));
+        } else {
+            module.content = None;
+            module.semi = Some(Semi::default());
```

Comment:
i think in `writer/rust/mod.rs` where we do `cxx_mod.content = Some(...)` when the `cxx_mod.content` is None. We should then do `cxx_mod.semi = None` to ensure we output the right thing?